cli/tools/repl/mod.rs

```diff
@@ -131,6 +131,7 @@ pub async fn run(
     match line {
       Ok(line) => {
         should_exit_on_interrupt = false;
+        editor.update_history(line.clone());
```

Comment:
Does this save `close()` in the history? Currently it doesn't. Should we?

Reply:
Yes it does; I think it's useful, it's a lot of chars to type :P